instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -154,17 +158,31 @@
     _DEFAULT_CONFIG.pop("system.network.connections")
 
 
+def _build_default_config() -> dict[str, list[str] | None]:
+    excluded_metrics: list[str] = [
+        pat.strip()
+        for pat in os.environ.get(
+            OTEL_PYTHON_SYSTEM_METRICS_EXCLUDED_METRICS, ""
+        ).split(",")
+        if pat
```

Comment:
```suggestion
        if pat.strip()
```

To avoid empty strings if there are spaces in the env var